docs/sources/flow/concepts/modules.md

```diff
@@ -26,7 +26,7 @@ Modules can be [imported](#importing-modules) to enable the reuse of [custom com
 A module can be _imported_, allowing the custom components defined by that module to be used by other modules, called the _importing module_.
 Modules can be imported from multiple locations using one of the `import` configuration blocks:
 
-* [import.file]: Imports a module from a file on disk.
+* [import.file]: Imports a module from a file or a folder on disk.
```

Comment:
Do we really mean `folder` here? or should this be `directory`?  My instinct is to say `directory` here and in the other instances.

Grafana defaults to the Google wordlist: https://developers.google.com/style/word-list#directory which says... it should be `directory`

Reply:
We mean directory cc @wildum, these should change to say "directory"